File: Palindrome Number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """

        x_reverse = "".join(reversed(str(x)))

        if (x_reverse == str(x)):
            return True
        else:
            return False
        

File: Palindrome Number.go
func isPalindrome(x int) bool {
	string := strconv.Itoa(x)
	l := len(string)
	for i := 0; i < l; i++ {
		if string[i] != string[l-1-i] {
			return false
		}
	}
	return true
}